meshroom/core/graph.py

```diff
@@ -234,7 +234,15 @@ def fileFeatures(self):
         return Graph.IO.getFeaturesForVersion(self.header.get(Graph.IO.Keys.FileVersion, "0.0"))
 
     @Slot(str)
-    def load(self, filepath):
+    def load(self, filepath, fileLink=True):
```

Comment:
`fileLink` is a bit obscure.
why not turn the table and use `asTemplate=False`. I know that then all the conditions should be negate but it is less obscure than fileLink IMO.

Reply:
I would prefer to avoid inverted negation: `asTemplate=False` will do more actions (setup the cacheDir, etc.) and `asTemplate=True` to do nothing.
I pushed a proposition for another name that doesn't negate the option: `setupFileRef`.
Does that sound fine to you?